### ISSUE TITLE
Livestreamed subtitles loopback on old feed

### ISSUE THREAD

**Issue by EdNutting** [clowdr-app/clowdr#25]
When two feeds are played one after the other in the same livestream and the second does not have embedded captions/subtitles, AWS seems to loop back on the previous video's subtitles.

We also need to go back and fix this for the existing Q&A recordings from POPL, which have ended up with incorrect subtitles embedded.

**Comment by EdNutting** [comment]
@rossng Did all this get resolved during the Playout service (re-)write?

**Comment by rossng** [comment]
It was never exactly clear what was causing this in the live playback. I don't think we have seen it again recently so I suggest we close this issue and reopen if it comes up again. It's quite possible that there was a bug in MediaPackage or hls.js that is now resolved.

The subtitles in recordings was a completely separate issue with a definite cause. That has been fixed for a while.
